fix(docs/workflow): adicionar fi faltante no step Update SPEC versions

## Problema

O workflow `.github/workflows/docs.yml` está falhando com erro de sintaxe bash:

``
Run VERSION=0.3.0
/home/runner/work/_temp/682d2b35-f8b1-4d0f-a02d-2589f1864424.sh: line 20: syntax error: unexpected end of file
Error: Process completed with exit code 2.
``

## Causa Raiz

No step "Update SPEC versions" (linhas 79-100), falta o `fi` para fechar o bloco `if/else` no final do script bash.

## Solução Proposta

Adicionar `fi` na linha 100, após o `echo "docs/spec directory does not exist, skipping"`.

## Contexto

- Workflow: docs.yml
- Step afetado: Update SPEC versions
- Trigger: push para main em docs/, VERSION ou workflow
- Erro reportado em: https://github.com/h4mn/skybridge/actions/runs/20901981494

## Referências

- ADR012: Single source of truth via arquivo VERSION
- Docs workflow integration

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -97,6 +97,7 @@ jobs:
             fi
           else
             echo "docs/spec directory does not exist, skipping"
+          fi
 
       - name: Generate version index
         run: |

